### DISCUSSION TITLE
关于防火墙功能需求建议

### DISCUSSION THREAD

**Discussion by LyingDoc** [1Panel-dev/1Panel#5012] [Ideas]
### 1Panel 版本

v1.10.7

### 请描述您的需求或者改进建议

![图片](https://github.com/1Panel-dev/1Panel/assets/131896721/6e4373a6-3bfa-4cb4-aa8d-0e9e22d560cc)
此功能收费不合理，

### 请描述你建议的实现方案

为网站访问这个收费功能建议开放出来，你怎么收费没啥意义，移出高级功能，作为免费，挂上线了，攻击ip太多，参照小皮面板开源人家也没有把这个功能收费，还有点，高级功能建议隐藏，等人激活了显示出来，没必要一直显示出来，点击人家也用不了，显示多余

**Comment by zhengkunwang223** [comment]
暂时不会更改
